tests/system/Files/FileCollectionTest.php

```diff
@@ -124,7 +142,7 @@ public function testConstructorCallsDefine()
         $collection = new class () extends FileCollection {
             protected function define(): void
             {
-                $this->add(SUPPORTPATH . 'Files/baker/banana.php');
+                $this->add(normalize_path(SUPPORTPATH . 'Files/baker/banana.php'));
```

Comment:
Do you write product code like this?

Reply:
I don't get it. Do you want me instead to normalize the `add` method? I mean do the logic of normalizing there?

Reply:
Probably I don't write code like that, and I don't want to recommend the code like that.
So it seems not good code even in test code.

I'm not sure normalizing in the `add()` is good or not.

Is it impossible to pass the test only normalizing the output?